docs/phylum.md

```diff
@@ -0,0 +1,37 @@
+---
+title: phylum
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Interact with the Phylum API.
+
+### Core Commands
+* [phylum analyze](https://docs.phylum.io/docs/phylum_analyze)
+* [phylum auth](https://docs.phylum.io/docs/phylum_auth)
+* [phylum projects](https://docs.phylum.io/docs/phylum_projects)
```

Comment:
```suggestion
* [phylum project](https://docs.phylum.io/docs/phylum_projects)
```
`phylum projects` does work...but as an alias. The current subcommand name is `project`...as can be seen in the output of a `phylum help` command.

Reply:
Ah yes, this happened after I had created this page. The link also needed to be changed, I have updated both.